python/ray/tests/test_actor.py

```diff
@@ -362,6 +362,29 @@ def __init__(self):
     assert actor_class_info[b"module"] == b"ray.tests.test_actor"
 
 
+def test_actor_inheritance(ray_start_regular):
+    class NonActorBase(object):
+        def __init__(self):
+            pass
+
+    # Test that an actor class can inherit from a non-actor class.
+    @ray.remote
+    class ActorBase(NonActorBase):
+        def __init__(self):
+            pass
+
+    # Test that you can't instantiate an actor class directly.
+    with pytest.raises(Exception):
+        ActorBase()
+
+    # Test that you can't inherit from an actor class.
+    with pytest.raises(TypeError):
```

Comment:
If you want to check the contents of the error message, I think you can do something like

```suggestion
    with pytest.raises(TypeError, match="Inheriting from actor classes is not currently supported."):
```

taken from https://doc.pytest.org/en/latest/assert.html

Reply:
I'd suggest doing this for both this one and the previous exception as well.